CONFIGURING-DETECTORS.md

```diff
@@ -0,0 +1,103 @@
+# Configuring Detectors for Edge Inference
+
+## Do I need to configure my detectors?
+
+Detector-specific configuration is NOT necessary for basic use of the edge endpoint. Once the edge endpoint is running, submitting an image query to it will create an edge inference pod for the specified detector. This inference pod will be created with the default settings. Once the inference pod has loaded the edge model and is ready to serve requests, it will begin attempting to answer image queries sent to that detector. If the answer from the edge model has confidence above the confidence threshold for the detector, you'll receive the answer from the edge model. If the confidence isn't high enough, it will escalate the query to the cloud. 
+
+## Why would I want to configure detectors?
+
+Configuring detectors for the edge endpoint allows you to provide fine-grained control over their behavior on the edge. By configuring a detector, you can:
+* Automatically create an inference pod for the detector every time the edge endpoint starts up, ensuring it's ready to serve requests without having to first submit a query to it
+* Configure the detector to always return the edge model's answer, regardless of the confidence
+* Configure the detector to never escalate queries to the cloud
+* Specify a maximum frequency for cloud escalations
```

Comment:
Might want to specify that this is done for a particular detector, not for the entire edge setup